plugin-server/src/utils/db/db.ts

```diff
@@ -308,7 +308,7 @@ export class DB {
                     // invalid JSON
                     return null
                 } else {
-                    throw error
+                    throw error // TODO: add a test for this making sure we still ingest the event if we hit an error querying redis for groups info
```

Comment:
From local testing with current code if we throw here for whatever reason, then the event ends up in DLQ - we probably want to query postgres instead for group props